### PR TITLE
Update the asset file path for Android Benchmark app setup

### PR DESCRIPTION
**Pull request by elviscapiaq**

### CODE REVIEW

File: benchmarks/graphics_pipeline/build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ArrayList projectAssets = [
     "benchmarks/shaders/spv",
+    "benchmarks/textures",
```
Comment (wangra-google) [inline benchmarks/graphics_pipeline/build.gradle:3]: there are some textures under the `Wood` folder, are they used by benchmarks? if so, should we add the subfolder? if not, shall we remove those textures? https://github.com/google/bigwheels/tree/main/assets/benchmarks/textures/Wood

Comment (angela28chen) [reply benchmarks/graphics_pipeline/build.gradle:3]: I think these are used by the benchmark `texture_transfer_cpu_to_gpu`

Comment (wangra-google) [reply benchmarks/graphics_pipeline/build.gradle:3]: good to know, thanks

Comment (elviscapiaq) [reply benchmarks/graphics_pipeline/build.gradle:3]: I believe when you add a folder. all files inside it and its subfolders are also added, so we shouldn't need to add the `Wood` subfolder.

**Comment by elviscapiaq** [comment]
I tested this last update on the Benchmark app and it is working as expected now.

**Review by wangra-google — APPROVED** [review]

**Comment by wangra-google** [comment]
> I tested this last update on the Benchmark app and it is working as expected now.

LGTM

**Review by angela28chen — APPROVED** [review]

**Review by apazylbe — APPROVED** [review]

**Comment by elviscapiaq** [comment]
Closing this PR since I created it over my local main branch and had several issues to rebase it.
I created a new PR with this same change: https://github.com/google/bigwheels/pull/335
I am sorry for this.

**Pull request closed without merging**